recipes/zeus-mcmc/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "zeus-mcmc" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b291e734ef65c05b2fb6c3f8867066133075f7a8e95c9489d21ae06062365693
+  patches:
+    # requirements and README.md aren't packaged
```

Comment:
Can you open a PR or issue upstream so that this gets fixed for the next release if it hasn’t been already?

Reply:
Yep, this has been fixed in the zeus repo now https://github.com/minaskar/zeus/pull/16, so the patch won't be necessary for the next release.